<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>444</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Easy Notepad</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icon.qrc">
    <normaloff>:/icon-image.png</normaloff>:/icon-image.png</iconset>
  </property>
  <property name="accessibleName">
   <string>Easy Notepad</string>
  </property>
  <property name="accessibleDescription">
   <string>Easy Notepad.</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SafeTextEdit" name="textEdit">
      <property name="accessibleName">
       <string>Easy Notepad</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::TextEditable|Qt::TextSelectableByKeyboard</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuKeyAndPassword">
     <property name="title">
      <string>Key and Password</string>
     </property>
     <addaction name="actionSet_Encryption_Key"/>
     <addaction name="actionChange_Password"/>
     <addaction name="action_Logout"/>
    </widget>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Email"/>
    <addaction name="separator"/>
    <addaction name="actionRename_File"/>
    <addaction name="action_Move_File"/>
    <addaction name="action_Delete"/>
    <addaction name="action_Undelete"/>
    <addaction name="separator"/>
    <addaction name="menuKeyAndPassword"/>
    <addaction name="separator"/>
    <addaction name="action_SetAsDefault"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menu_Search">
    <property name="title">
     <string>&amp;Search</string>
    </property>
    <addaction name="action_Find"/>
    <addaction name="actionFind_Next"/>
    <addaction name="separator"/>
    <addaction name="actionFind_Global"/>
    <addaction name="actionFind_Old"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Insert_Template"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_Help"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Search"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>32</height>
    </size>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
  </widget>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Delete">
   <property name="text">
    <string>&amp;Delete File</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="action_Find">
   <property name="text">
    <string>&amp;Find First</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionFind_Next">
   <property name="text">
    <string>Find &amp;Next</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFind_Global">
   <property name="text">
    <string>Find &amp;Global</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionFind_Old">
   <property name="text">
    <string>Find Global (include &amp;All)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+G</string>
   </property>
  </action>
  <action name="action_Undelete">
   <property name="text">
    <string>&amp;Undelete File</string>
   </property>
  </action>
  <action name="actionRename_File">
   <property name="text">
    <string>&amp;Rename File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_Move_File">
   <property name="text">
    <string>&amp;Move File</string>
   </property>
  </action>
  <action name="action_Insert_Template">
   <property name="text">
    <string>&amp;Insert Template</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Copy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Shift+Del</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_Help">
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_Email">
   <property name="text">
    <string>&amp;Attach as Email</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSet_Encryption_Key">
   <property name="text">
    <string>Set Encryption Key</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>Change &amp;Password</string>
   </property>
  </action>
  <action name="action_Logout">
   <property name="text">
    <string>&amp;Logout</string>
   </property>
  </action>
  <action name="action_SetAsDefault">
   <property name="text">
    <string>Set as Default for File Types</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SafeTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>../Lib/safetextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Icon.qrc"/>
 </resources>
 <connections/>
</ui>
